RomanNumeralsConverter.Tests/RomanNumeralValidatorManagerTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using Rhino.Mocks;
+
+namespace RomanNumeralsConverter.Tests
+{
+  internal class RomanNumeralValidatorManagerTests
```

Comment:
Just few test cases here. It look like you do not use TDD here. For instance it is lack of test case when rules are null